code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -250,6 +251,8 @@ GLOBAL_LIST_INIT(cloth_recipes, list ( \
 	new/datum/stack_recipe("black gloves", /obj/item/clothing/gloves/color/black, 3), \
 	null, \
 	new/datum/stack_recipe("blindfold", /obj/item/clothing/glasses/sunglasses/blindfold, 2), \
+	new/datum/stack_recipe("muzzle", /obj/item/clothing/mask/muzzle, 2), \
```

Comment:
/obj/item/clothing/mask/muzzle/gag
Doesn't have to be anything different, but sets up for a new icon_state if you want a cloth gag.